src/backend/users/migrations/0004_add_location_to_destination.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.13 on 2022-05-01 18:21
+
+import django.contrib.gis.db.models.fields
+import django.contrib.gis.geos.point
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0003_create_destination'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='destination',
+            name='address',
+        ),
+        migrations.AddField(
+            model_name='destination',
+            name='location',
+            field=django.contrib.gis.db.models.fields.PointField(help_text='The lat/lng point location of the destination', srid=4326),
```

Comment:
This migration failed for me because I already had Destinations that had been added to my development database. When the migration ran, those Destinations ended up with null locations, which violated the not-null constraint. There's not really a great solution here because we have no way of knowing what location the existing Destinations should have, but luckily this only impacts developers so I think it should be safe to simply use the coordinates for Boston:
```suggestion
            field=django.contrib.gis.db.models.fields.PointField(default=Point(-71.06, 42.36, srid=4326), help_text='The lat/lng point location of the destination', srid=4326),
```

(You'll also need to add `from django.contrib.gis.geos import Point` up top)

To test this you can migrate back to 0003, add a UserProfile with Destinations using the Django admin, and then attempt to migrate forward.

Reply:
Ohh, oops, agreed. Lemme add it!